rules/90-hda-powersave.rules

```diff
@@ -0,0 +1,11 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="hda_powersave_end"
+
+SUBSYSTEM=="sound", DRIVERS=="snd_hda_intel", \
+  IMPORT{builtin}="hwdb 'snd_hda_intel:$attr{subsystem_vendor}:$attr{subsystem_device}:$attr{[dmi/id]modalias}'"
+
+ENV{SND_HDA_INTEL_POWERSAVE}!="", \
+  RUN+="/bin/sh -c 'echo $env{SND_HDA_INTEL_POWERSAVE} > /sys/module/snd_hda_intel/parameters/power_save'"
```

Comment:
hrmpf.. so we alter per-driver switches based on devices found. This means a system with two hda sound cards will get quite an undeterministic behaviour here, no?